assets/lib/share-link/share-link.js

```diff
@@ -116,22 +106,20 @@
 		height: 480,
 	};
 
-	ShareLink.getNetworkLink = function( networkName, settings ) {
-		var link = ShareLink.networkTemplates[ networkName ].replace( /{([^}]+)}/g, function( fullMatch, pureMatch ) {
+	window.ShareLink.getNetworkLink = function( networkName, settings ) {
```

Comment:
For clarification: why do we have to add the `window` object here?

Maybe arrow function?